packages/styles/_variables.scss

```diff
@@ -29,6 +29,11 @@ $colors: (
   green-400: #19a654,
   green-500: #008036,
 
+  // Creation button - discuss with designer
+  green-creation-1: #008871,
```

Comment:
Why it does not follow same convention as the others? Like naming `green-600` and `green-700`?

Reply:
The discussion about this palette is in progress.

Reply:
We are falling back to the existing colors while waiting for the new accessibility-friendly palette